Doc/tutorial/errors.rst

```diff
@@ -101,29 +101,29 @@ The :keyword:`try` statement works as follows.
 * If no exception occurs, the *except clause* is skipped and execution of the
   :keyword:`try` statement is finished.
 
-* If an exception occurs during execution of the try clause, the rest of the
+* If an exception occurs during execution of the *try clause*, the rest of the
   clause is skipped.  Then if its type matches the exception named after the
-  :keyword:`except` keyword, the except clause is executed, and then execution
-  continues after the :keyword:`try` statement.
+  :keyword:`except` keyword, the *except clause* is executed, and then execution
+  continues after the *try/except* block.
```

Comment:
Try/except is an appropriate name for the block, but it is not a literal keyword in the code.
```suggestion
  continues after the try/except block.
```